handlers/main.yml

```diff
@@ -1,14 +1,15 @@
 ---
+
```

Comment:
NB reorder was necessary here as now slurmd needs to be able to contact slurmctld to start (as that's how it gets the config)